packaging/grafana-agent/windows/install_script.nsis

```diff
@@ -132,6 +132,8 @@ Function Install
     Pop $0
     nsExec::ExecToLog `sc start "Grafana Agent"`
     Pop $0
+    # Auto-restart agent on failure. Reset failure counter after 60 seconds without failure
+    nsExec::ExecToLog `sc failure "Grafana Agent" reset= 60 actions= restart/5000 reboot= "Grafana Agent has failed. Restarting in 5 seconds"`
```

Comment:
Can you add this to the Flow NSIS script too?

Reply:
Oh there's another one? Let me dig around. Thanks for the lead!

Reply:
Yeah, it's right here :) https://github.com/grafana/agent/blob/main/packaging/grafana-agent-flow/windows/install_script.nsis

Reply:
Also, I think this needs a `Pop $0` after it (like in the ExecToLog above) to pop the result off the stack.

Reply:
Gotcha.

Reply:
`Pop $0` added and similar configs has been added for Flow agent's Windows installer in latest commit. PTAL. Thanks!